src/Extensions/SiteTreeExtension.php

```diff
@@ -1,26 +0,0 @@
-<?php
-
-namespace DNADesign\Elemental\Extensions;
-
-use DNADesign\Elemental\Models\ElementalArea;
-use SilverStripe\ORM\DataExtension;
-
-/**
- * The SiteTreeExtension provides a consistent API surface for reading elements from pages regardless
- * of whether the specific implementation of SiteTree (subclass) has the elemental extension applied.
- */
-class SiteTreeExtension extends DataExtension
-{
-    /**
-     * Returns the owner's `ElementalArea()` contents if it has the extension, or an empty one
-     *
-     * @return ElementalArea
-     */
-    public function ElementalAreaIfExists()
-    {
-        if ($this->owner->hasExtension(ElementalPageExtension::class)) {
-            return $this->owner->ElementalArea();
-        }
-        return ElementalArea::create();
-    }
-}
```

Comment:
Also needs the config removed: https://github.com/dnadesign/silverstripe-elemental/blob/ed298dc8c712761154c16760f61572b03c6ccfbb/_config/extensions.yml

Reply:
I did remove that :) It's at the very top of the diff.

Reply:
Weird, I found it in your commit tree. I guess the dev/build would've failed anyway, ignore me